### ISSUE TITLE
auto complete problem's solution in cpp

### ISSUE THREAD

**Issue by anjalisoni3655** [amanmehara/programming#78]
I would like to contribute to this issue during the hacktoberfest. Please assign this issue to me. Thanks.

**Comment by AkshadK7** [comment]
Can I be assigned to work on some Linked Lists problems for this issue?

Coz I don't see any Linked Lists implementations in C++ in your Repository.

P.S: Thanks in Advance :)

**Comment by amanmehara** [comment]
@AkshadK7 
Yes, you can work on creating linked list implementation in C++. 😃 

Create a PR once you are done with the changes.

**Comment by AkshadK7** [comment]
Sure !

**Comment by utkarshbajaj** [comment]
Can I work on implementing Dijkstra, Prims and Kruskal's algorithms in graphs?

**Comment by Dhruv-VINT** [comment]
I would also like to add a few algorithms to this repo, can you assign me too?

**Comment by amanmehara** [comment]
@Dhruv-VINT simply create a PR with the algorithms you want to contribute.
I'll review and merge them!

**Comment by bobbysaprey420** [comment]
I would like to contribute please assign something to me

**Comment by amanmehara** [comment]
@bobbysaprey420 
Yes, you can work on creating linked list implementation in C++. 😃

Create a PR once you are done with the changes.
(Also have a look at: https://github.com/amanmehara/programming/blob/master/CONTRIBUTING.md)

**Comment by TheShubham-K** [comment]
Can I be assigned to work on some more problems in competitive programming for this issue?

P.S: Thanks in Advance :)

**Comment by bobbysaprey420** [comment]
yes sure

**Comment by TheShubham-K** [comment]
okay thanks a lot

**Comment by bobbysaprey420** [comment]
@amanmehara is their anything I can contribute to earn some PR for Hacktoberfest.
Thanks in Advance
